app/views/teacher_mailer/application_declined.text.erb

```diff
@@ -19,29 +19,27 @@ Thank you for applying for QTS in England. Unfortunately, your application has b
 <% unless @view_object.declined_cannot_reapply? || @view_object.from_ineligible_country? %>
 # What you can do next
 
-You’ll be able to make a new application for QTS once you've addressed the decline reasons we've given. If you reapply for QTS, you’ll not be able to review the details of your previous application.
+You’ll be able to make a new application once you've addressed all decline reasons given. If you reapply for QTS, you’ll not be able to review the details of your previous application.
 
-[Apply again](<%= new_teacher_interface_application_form_url %>)
+You may want to explore [other routes to getting QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
 
-You may want to [explore other routes to getting QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
+[Apply again](<%= new_teacher_interface_application_form_url %>)
 
 <% end %>
 <% if @view_object.from_ineligible_country? %>
-[Sign in](<%= new_teacher_session_url %>) for more information about QTS in England.
-<% else %>
 # Decision review
 
-Applicants who have been declined for QTS are entitled to a review of the decline decision by a Casework Manager.
-If you would like to request a decision review, you’ll need to provide:
+Applicants who have been declined for QTS are entitled to a review of the decline decision by a Casework Manager. 
+
+If you would like to request a decision review, you will need to provide:
 
 - formal evidence and reasoning as to how you meet the required assessment criteria
 - additional information not included in your original application that would support your decision review
 
-Your request for review must be received within 28 days of receipt of the decision to decline QTS. Email your request for review, including the information required as above, to [<%= t("service.email.enquiries") %>](mailto:<%= t("service.email.enquiries") %>).
+Your request for review must be received within 28 days of receipt of the decision to decline QTS.
 
-If you are not satisfied with the outcome of the decision review, you can request a final formal appeal to a senior Teacher Qualification Manager.
-
-[Sign in](<%= new_teacher_session_url %>) to review the outcome of your application.
 <% end %>
 
+Email your request for review, including the information required as above to [<%= t("service.email.enquiries") %>](mailto:<%= t("service.email.enquiries") %>).
```

Comment:
I think this line belongs within the `if` block as it relates to the "Decision review" section.